pallets/subtensor/src/uids.rs

```diff
@@ -9,6 +9,22 @@ impl<T: Config> Pallet<T> {
         SubnetworkN::<T>::get(netuid)
     }
 
+    pub fn set_emission_for_uid(netuid: u16, neuron_uid: u16, emission: u64) {
+        Emission::<T>::mutate(netuid, |v| v[neuron_uid as usize] = emission);
+    }
+    pub fn set_trust_for_uid(netuid: u16, neuron_uid: u16, trust: u16) {
+        Trust::<T>::mutate(netuid, |v| v[neuron_uid as usize] = trust);
+    }
```

Comment:
Instead of creating a layer of indirection, I would just do the `::mutate` operations inline where they are used. Makes it faster to understand what the code is doing, and doesn't add any crazy verbosity or anything.